frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNoteEditor.tsx

```diff
@@ -0,0 +1,288 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```

Comment:
Nit: This will be fixed in my PR (#550) but would you mind updating these to 2017-2021 (as far as we know, it should be 2017-$CURRENT_YEAR)